src/Styleguide/Components/FollowArtistPopover/index.tsx

```diff
@@ -0,0 +1,87 @@
+import { FollowArtistPopover_suggested } from "__generated__/FollowArtistPopover_suggested.graphql"
+import { FollowArtistPopoverQuery } from "__generated__/FollowArtistPopoverQuery.graphql"
+import { ContextConsumer, ContextProps } from "Artsy/SystemContext"
+import React from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import styled from "styled-components"
+import { FollowArtistPopoverRowFragmentContainer as FollowArtistPopoverRow } from "./FollowArtistPopoverRow"
+
+const Container = styled.div``
+
+interface Props extends ContextProps {
+  suggested: FollowArtistPopover_suggested
+}
+
+class FollowArtistPopover extends React.Component<Props, null> {
+  render() {
+    const { suggested, user } = this.props
+    const { related } = suggested
+    return (
+      <Container>
+        {related.suggested.edges.map(({ node: artist }) => {
+          return (
+            <FollowArtistPopoverRow
+              user={user}
+              key={artist.__id}
+              artist={artist}
+            />
+          )
+        })}
+      </Container>
+    )
+  }
+}
+
+export const FollowArtistPopoverFragmentContainer = createFragmentContainer(
+  FollowArtistPopover,
+  graphql`
+    fragment FollowArtistPopover_suggested on Artist {
+      related {
+        suggested(first: 3, exclude_followed_artists: true) {
+          edges {
+            node {
+              __id
+              ...FollowArtistPopoverRow_artist
+            }
+          }
+        }
+      }
+    }
+  `
+)
+
+export const FollowArtistPopoverQueryRenderer = ({
```

Comment:
It's very possible this won't have its own query renderer when being used along with the Follow button on an existing/new page. But for now (+ storybooks), we'll include it.